packages/cli/src/commands/setup/ui/templates/storybook.preview.js.template

```diff
@@ -2,12 +2,12 @@ import * as React from 'react'
 
 import { ChakraProvider } from '@chakra-ui/react'
 
-const withChakra = (StoryFn) => {
+const WithChakra = (StoryFn) => {
```

Comment:
I wasn't going to comment, but now that this will clash with #4900 I think we better agree on the correct casing here. And that depends on how this is used, which I'm not familiar with.

If this is used like a component, i.e.
```
<WithChakra>
  <FooComponent />
</WithChakra>
```
then it should be PascalCase

If it's used like a HOC, i.e.
```
const FooComponent = () => <p>foo</p>
export default withChakra(FooComponent)
```
then it should be camelCase

I think it's a HOC, and then I'd prefer to keep this in camelCase.

Reply:
I believe is a component (but HOC is also a component ?) and I got this error `React component must start with an uppercase letter` let me check on Storybook docs.

Reply:
https://reactjs.org/docs/higher-order-components.html
The React docs also use cameCase for HOCs.

Reply:
Okay, for this case I really don't know, but if with we use camelCase we need to add ` // eslint-disable-next-line react-hooks/rules-of-hooks`

Reply:
... but we are in the `withI18n` HOC... Interesting!